.gitpod.yml

```diff
@@ -0,0 +1,32 @@
+image:
+  file: .gitpod.dockerfile
+
+github:
+  prebuilds:
+    branches: true
+
+ports:
+- port: 3000
+  onOpen: ignore
+# We open this port because it's the port that nginx is serving and which is federating the other services (web app and chain in this case)
+- port: 8000
+  onOpen: open-preview
+- port: 8900
+  onOpen: ignore
+- port: 8080
+  onOpen: ignore
+- port: 8888
+  onOpen: ignore
+- port: 9876
+  onOpen: ignore
+
+tasks:
+- name: WebApp
+  init: yarn
+  # We need the $PWD here because this path is not resolved from the current folder but from the path nginx runs from.  $PWD is the workspace path where the copy of the repo files is
```

Comment:
Nitpicky but `$PWD is the workspace path where the copy of the repo files` **is** -> `$PWD is the workspace path where the copy of the repo files` **are**